examples/tourism/package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "media",
+  "name": "tourism",
   "version": "1.0.0",
   "private": true,
   "dependencies": {
     "algoliasearch": "3.33.0",
     "qs": "6.8.0",
     "react": "16.8.6",
     "react-dom": "16.8.6",
-    "react-instantsearch-dom": "5.7.0"
```

Comment:
diff is going a big weird here, but seems correct

Reply:
yeah, git mixed tourism and media `package.json` files :neutral_face: